.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "import/extensions": [ 1, {
+      "js": "always", "json": "always"
+    }]
+  },
```

Comment:
- [ ] Please fix this error I see when I run your project : 
![image](https://user-images.githubusercontent.com/74744787/181838656-4830d23d-5a6f-4147-8515-5a3696d0271c.png)

Reply:
[MeqdamAlqudah](https://github.com/MeqdamAlqudah) I don't have this error on my console. I'm not sure where you're seeing it. Kindly consider that it might be an issue that is specific to your machine. This is the view on my console.

![image](https://user-images.githubusercontent.com/40690789/181842076-c280bb6b-c9cd-4fda-9734-953957345b75.png)

Reply:
Fixed